release-1.8/release_notes_draft.md

```diff
@@ -165,68 +165,43 @@ across the system. Here's the release [scalability validation report].
-
-* CronJobs has been promoted to `v1beta1` which is now turned on by default.
-  Although version `v2alpah1` is still available, it is deprecated.  Migrate to
-  `batch/v1beta1.CronJobs`.  Additionally, upgrading cluster in high-availability
-  configuration might return errors. The new controllers rely on the latest
-  version of the resources.  If the expected version is not found during rolling
-  upgrade, the system throws resource not found errors.
-* The `batch/v2alpha1.ScheduledJobs` has been removed.  Migrate to `batch/v1beta.CronJobs`
-  to continue managing time based jobs.
-
-* The APIs `rbac/v1alpha1`, `settings/v1alpha1`, and `scheduling/v1alpha1` are
-  disabled by default.
-
-* The `system:node` role is no longer automatically granted to the `system:nodes`
-  group in new clusters. It is recommended that nodes be authorized using the
-  `Node` authorization mode instead. Installations that wish to continue giving
-  all members of the `system:nodes` group the `system:node` role (which grants
-  broad read access, including all secrets and configmaps) must create an
-  installation-specific `ClusterRoleBinding`. ([#49638](https://github.com/kubernetes/kubernetes/pull/49638))
+  Don't use these flags. Using deprecated flags causes the server to print a warning. Using a removed flag causes the server to abort the startup.
+
+* StatefulSet: The deprecated `pod.alpha.kubernetes.io/initialized` annotation for interrupting the StatefulSet Pod management is now ignored. If the annotation is set to `true` or left blank, you will not see any change in behavior. If the annotation is set to `false`, previously dormant StatefulSets might become active after upgrading.
+
+* CronJobs was moved to `v1beta1` and is turned on by default. The deprecated `v2alpah1` CronJobs is still available, but it must be explicitly turned on. It's recommended that you migrate the current CronJobs to `batch/v1beta1.CronJobs`. The version changes might introduce errors during cluster upgrades in high-availability configuration. The new controllers rely on `v1beta1` of the resources during rolling upgrade. If the expected version is not found, the system throws resource not found errors.
+
+* The `batch/v2alpha1.ScheduledJobs` was removed. Migrate to `batch/v1beta.CronJobs` to continue managing time based jobs.
+
+* The `rbac/v1alpha1`, `settings/v1alpha1`, and `scheduling/v1alpha1` APIs are disabled by default.
+
+* The `system:node` role is no longer automatically granted to the `system:nodes` group in new clusters. The role gives broad read access to resources, including secrets and configmaps. Use the `Node` authorization mode to authorize the nodes in new clusters. To continue providing the `system:node` role to the members of the `system:nodes` group, create an installation-specific `ClusterRoleBinding` in the installation. ([#49638](https://github.com/kubernetes/kubernetes/pull/49638))
 
```

Comment:
If we're going to specify objects in code markup, we need to be careful that they match the exact object name. Here it should be `batch/v2alpha1.ScheduledJob` object, for example, and then the same for CronJob.